docs/topics/mta-web-edit-application.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-edit-application_{context}"]
+= Opening and editing an application
+
+You open and edit an existing application to the *Application Inventory* for changes in the assessment and analysis.
+
+.Prerequisites
+
+* Log in to a server running {ProductShortName}
+
+.Procedure
+
+. In *Migration* view, click *Application Inventory*.
+. Click the row of an application to open a summary of the application's current settings.
++
+.View and edit application
+image::mta-web-app-details-01.png[View and edit application]
+
+. To change application settings, click `edit` icon {icon-edit} to open the application settings, and review the application settings. See previous procedure for a list of application settings.
```

Comment:
Can we link to the previous procedure here?

Reply:
Added note in beginning of procedure